README.md

```diff
@@ -1,56 +1,20 @@
 [![CircleCI](https://circleci.com/gh/signalfx/signalfx-go-tracing/tree/master.svg?style=svg)](https://circleci.com/gh/signalfx/signalfx-go-tracing/tree/master)
 [![GoDoc](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing?status.svg)](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing)
 
-### Installing
-Add `github.com/signalfx/signalfx-go-tracing` to your `go mod` or `dep` dependencies.
+# SignalFx Tracing Library for Go
 
-Requires:
+The SignalFx Tracing Library for Go helps you instrument Go applications
+with the OpenTracing API to capture and report distributed traces to
+SignalFx.
```

Comment:
Since it provides a tracer and instrumented library helpers I think this can be expanded a bit.  How I read this is "the library helps you manually instrument using the OT api", which isn't entirely the case (though true).